src/bootstrap/bootstrap.py

```diff
@@ -881,8 +881,9 @@ def update_submodules(self):
         submodules_names = []
         for module in submodules:
             if module.endswith("llvm-project"):
-                if self.get_toml('llvm-config') and self.get_toml('lld') != 'true':
-                    continue
+                if self.get_toml('llvm-config') or self.get_toml('download-ci-llvm') == 'true':
+                    if self.get_toml('lld') != 'true':
```

Comment:
Hm I don't know that we actually support lld with download-ci-llvm. I guess I didn't test :)

But seems okay for now until we can check.